src/v2/Apps/Auctions/auctionsRoutes.tsx

```diff
@@ -5,34 +5,89 @@ const AuctionsApp = loadable(() => import("./AuctionsApp"), {
   resolveComponent: component => component.AuctionsAppFragmentContainer,
 })
 
+const CurrentAuctions = loadable(() => import("./Routes/CurrentAuctions"), {
+  resolveComponent: component => component.CurrentAuctionsFragmentContainer,
+})
+
+const UpcomingAuctions = loadable(() => import("./Routes/UpcomingAuctions"), {
+  resolveComponent: component => component.UpcomingAuctionsFragmentContainer,
+})
+
+const PastAuctions = loadable(() => import("./Routes/PastAuctions"), {
+  resolveComponent: component => component.PastAuctionsFragmentContainer,
+})
+
 export const auctionsRoutes = [
   {
     path: "/auctions2",
     getComponent: () => AuctionsApp,
     prepare: () => {
       AuctionsApp.preload()
     },
+
     query: graphql`
       query auctionsRoutes_AuctionsQuery {
         me {
           ...AuctionsApp_me
         }
-        currentSales: salesConnection(
-          first: 99
-          published: true
-          sort: START_AT_ASC
-        ) {
-          ...AuctionsApp_currentSales
-        }
-        pastSales: salesConnection(
-          first: 20
-          published: true
-          live: false
-          sort: START_AT_ASC
-        ) {
-          ...AuctionsApp_pastSales
-        }
       }
     `,
+    children: [
```

Comment:
Can you elaborate a bit more as to the purpose of children in a GraphQl query? Is this returning the following selected data as children elements of `AuctionsApp_me`?

Reply:
(See responses about routing above, lemme know if that makes more sense...)